openapi/openapi.yaml

```diff
@@ -1312,3 +1353,21 @@ components:
         application/json:
           schema:
             $ref: '#/components/schemas/UsersPage'
+
+  securitySchemes:
+    BasicAuth:
+      type: http
+      scheme: basic
+      description: Username and password login.
+
+    ApiKeyAuth:
+      type: apiKey
+      in: header
+      name: x-rh-identity
+      description: Identity header provided by 3scale (for non-prod testing only).
+
+    ApiKeyAuthInternal:
```

Comment:
What's the difference between `ApiKeyAuth` and `ApiKeyAuthInternal`. They're literally the same.